### PR TITLE
Vets json schema package

### PR DESCRIPTION
**Pull request by lihanli**

### CODE REVIEW

**Comment by jbalboni** [comment]
@lihanli @akainic just put up a PR (#4024) to make an update to the schema. I want to make sure the changes she's adding are also in this PR.

**Comment by lihanli** [comment]
they're on the same commit

File: README.md [hunk 1]
```diff
@@ -31,7 +31,6 @@ very secret.
 | run automated accessibility tests | `npm run build && npm run test:accessibility` |
 | test for broken links | Build the site. Broken Link Checking is done via a Metalsmith plugin during build. Note that it only runs on *build* not watch. |
 | add new npm modules | `npm install -D my-module` followed by `npm shrinkwrap --dev`. There are no non-dev modules here. |
-| get the latest json schema | `git submodule init; git submodule update; cd schema; git checkout master; git pull; cd ..`. Then you can commit the changes, if any.  |
```
Comment (jbalboni) [inline README.md]: Can you put instructions in there on how to update the schema now that it's an npm package?

**Review by jbalboni — APPROVED** [review]
LGTM, just had one question about documentation.

**Pull request merged**